.github/workflows/full-integration-tests.yaml

```diff
@@ -49,10 +53,24 @@ jobs:
         name: Calculate Matrix for testsuite
         working-directory: dev/tests/integration
         run: |
+
+          ### TODO: rebuild all that hell to node-js
+          
+          if [ -n "${{ github.event.inputs.test_directory }}" ]; then
+              echo "testsuite_dirs=['${{ github.event.inputs.test_directory }}']" >> "$GITHUB_OUTPUT"
+              exit 0
+          fi
+          
           OUTPUT_FILE="integration-testsuites.json"
           TESTSUITE_DIR="./testsuite/Magento"
           CODE_DIR="../../../app/code"
-          MAX_DIRS_PER_LINE=15
+          MAX_DIRS_PER_LINE=1
+          
+          ## variable with array of exclusion list of directories - they will be handled separately and splitted on more batches
+          EXCLUSION_LIST=("./testsuite/Magento/Catalog" "./testsuite/Magento/Bundle" "./testsuite/Magento/AsynchronousOperations" "./testsuite/Magento/Sales")
+
+          ## variable with array of exclusion list of files - separate run will be executed for each file (separate place in tests matrix)
+          STANDALONE_TESTS=("./testsuite/Magento/AsynchronousOperations/Model/MassScheduleTest.php" "./testsuite/Magento/Catalog/Observer/SwitchPriceAttributeScopeOnConfigChangeTest.php")
```

Comment:
This seems to assume the action is used to run the Magento testsuite.
The action is intended to be able to run any set of integration tests in a Magento project, so for example tests in a module with a different namespace.
If I read the code correctly, the files listed in `STANDALONE_TESTS` will always be attempted to be executed.
Is that correct?